src/mon/MDSMonitor.cc

```diff
@@ -1581,6 +1582,15 @@ int MDSMonitor::management_command(
       return -ENOENT;
     }
 
+    string force;
+    cmd_getval(g_ceph_context,cmdmap, "force", force);
+    int64_t metadata_num_objects = mon->pgmon()->pg_map.pg_pool_sum[metadata].stats.sum.num_objects;
+    if (force != "--force" && metadata_num_objects > 0) {
+      ss << "pool '" << metadata_name
+	 << "' already contains some objects. Use another pool which is empty.";
```

Comment:
a native english speaker please correct me, but this phrase feels funny. I'd personally go with "Use an empty pool instead", or something of the sorts. The `which is empty` portion makes me all tingly (I should probably see a doctor about that...)